spec/ConfigChanges/ConfigFile.spec.js

```diff
@@ -19,73 +19,83 @@ var rewire = require('rewire');
+                configFile.prototype.save();
+                expect(fs.writeFileSync).toHaveBeenCalled();
+            });
+        });
     });
 
-    it('ConfigFile_save/ConfigFile.prototype.save', function () {
-        spyOn(fs, 'writeFileSync');
-        configFile.prototype.save();
-        expect(fs.writeFileSync).toHaveBeenCalled();
-    });
+    describe('static methods', () => {
+        describe('isBinaryPlist', () => {
+            it('should return false if not binary', function () {
+                spyOn(fs, 'readFileSync').and.returnValue('not bplist');
+                expect(configFile.isBinaryPlist('someFile')).toBe(false);
+            });
 
-    it('isBinaryPlist should return false if not binary', function () {
-        spyOn(fs, 'readFileSync').and.returnValue('not bplist');
-        expect(configFile.isBinaryPlist('someFile')).toBe(false);
-    });
-    it('isBinaryPlist should return true if binary', function () {
-        spyOn(fs, 'readFileSync').and.returnValue('bplist');
-        expect(configFile.isBinaryPlist('someFile')).toBe(true);
-    });
+            it('should return true if binary', function () {
+                spyOn(fs, 'readFileSync').and.returnValue('bplist');
+                expect(configFile.isBinaryPlist('someFile')).toBe(true);
+            });
+        });
 
-    it('getIOSProjectname should throw error', function () {
-        expect(function () { configFile.getIOSProjectname('some/project/name'); }).toThrow();
-    });
+        describe('getIOSProjectname', () => {
+            it('should throw error', function () {
+                expect(function () { configFile.getIOSProjectname('some/project/name'); }).toThrow();
+            });
+        });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var filePath = path.join('project_dir', 'file');
-        expect(configFile.resolveConfigFilePath('project_dir', 'platform', 'file')).toBe(filePath);
-    });
+        describe('resolveConfigFilePath', () => {
+            const projectDir = path.join('project_dir', 'app', 'src', 'main');
 
-    it('resolveConfigFilePath should return file path', function () {
-        var androidManifestPath = path.join(projectDir, 'AndroidManifest.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', 'AndroidManifest.xml')).toBe(androidManifestPath);
-    });
+            it('should return file path', function () {
+                var filePath = path.join('project_dir', 'file');
+                expect(configFile.resolveConfigFilePath('project_dir', 'platform', 'file')).toBe(filePath);
+            });
 
-    it('resolveConfigFilePath should return file path', function () {
-        var configPath = path.join(projectDir, 'res', 'xml', 'config.xml');
-        expect(configFile.resolveConfigFilePath('project_dir', 'android', 'config.xml')).toBe(configPath);
-    });
+            it('should return file path', function () {
```

Comment:
We already have a test `should return file path ` in this structure.
Could it be updated to provide some uniqueness? e.g. `should return AndroidManifest.xml  file path`

Reply:
```suggestion
            it('should return AndroidManifest.xml file path', function () {
```